tests/utils.py

```diff
@@ -134,17 +134,6 @@ def get_parameter_status(self, parameter):
         else:
             raise NotImplementedError()
 
```

Comment:
style check failed because this needs to be 2 blank lines

Reply:
Thanks, fixed and committed